catalog/nsili/catalog-nsili-source/src/main/java/org/codice/alliance/nsili/source/NsiliSource.java

```diff
@@ -316,12 +322,15 @@ private void initOrb() {
      * Initializes the Root STANAG 4559 Library Interface
      */
     private void initLibrary() {
-        org.omg.CORBA.Object obj = orb.string_to_object(iorString);
-        library = LibraryHelper.narrow(obj);
-        if (library != null) {
-            LOGGER.debug("{} : Initialized Library Interface", getId());
-        } else {
-            LOGGER.error("{} : Unable to initialize the library interface.", getId());
+        if (iorString != null) {
```

Comment:
Can iorString be null here?  Doesn't look like this method will be called if iorString is null https://github.com/connexta/alliance/pull/23/files#diff-e438597cbff79a6c527c262a3174df85R274